### PR TITLE
Provide the projects maturity

### PR DESCRIPTION
**Pull request by balhar-jakub**
Signed-off-by: Jakub Balhar <jakub.balhar@broadcom.net>

### CODE REVIEW

File: Technical-Steering-Committee/projects.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+
+#### Recommended Use
+
+Remove the project from the Production environment as the project will be removed from Zowe. 
+
+#### Possible transitions
+
+- From Deprecated to Archived means that the project development and support won’t continue
+
+### Archived
```
[the remark this answers is not in the document]
Comment (balhar-jakub) [reply Technical-Steering-Committee/projects.md:145]: Sure, it's a good point, though Zowe On The Go was never officially archived.

File: Technical-Steering-Committee/projects.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+
+- From Generally Available to Technical Preview means that we found out, based on the customer feedback, that the project needs a major rework for a longer life 
+- From Generally Available to Long Term Support means that the project is stable and mature. It belongs to the production 
+- From Generally Available to Archived means that the project wasn't a good fit and, as such, won't be further developed
+
+### Long Term Support
+
+#### Purpose
+
+The most stable version doesn't necessarily contain the latest and newest functionality, but it's ready to be deployed to the production environments. The Long Term Support is the most stable and longest supported version of any project. There are no breaking changes allowed within the major release.
```
Comment (ifakhrutdinov) [inline Technical-Steering-Committee/projects.md:103]: > There are no breaking changes allowed within the major release.

Are major releases possible with LTS? That is, can something go from LTS v2 to LTS v3 directly? Or would that be GA v3 first and then LTS v3. If it's the latter, can we just say "There are no breaking changes allowed." ?

File: Technical-Steering-Committee/projects.md [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+
+- From Technical Preview to Generally Available means that the project received the needed feedback and is ready to be put into production
+- From Technical Preview to Long Term Support means that the project is stable and mature. It belongs to the production
+- From Technical Preview to Archived means that after the feedback from the customers, the project found that it's not a good fit and the development won't continue
+
+### Generally Available
+
+#### Purpose
+
+To gather more feedback and improve the project before stabilizing it for Long Term Support. The project is stable, and we don't expect major changes, but unlike for Long Term Support, the changes may be needed. We believe that it's ok for GA products to become part of the production environments. The project may need to release major versions with breaking changes more often than the projects in Long Term Support. There should be no breaking changes in the minor releases.
```
Comment (ifakhrutdinov) [inline Technical-Steering-Committee/projects.md:79]: Would breaking changes include breaking changes in the source which would **not** affect runtime, e.g. 3rd party plugin would still work.

File: Technical-Steering-Committee/projects.md (hunk 2 above)
Comment (balhar-jakub) [reply Technical-Steering-Committee/projects.md:103]: Yes it's possible to go from LTS to LTS. 
If project doesn't feel the need to have a GA lifecycle stage it's ok.

File: Technical-Steering-Committee/projects.md (hunk 3 above)
Comment (balhar-jakub) [reply Technical-Steering-Committee/projects.md:79]: I would say that this level of detail, doesn't really belong here. There is a proposal for policy that explains in detail what breaking change mean here: https://github.com/zowe/community/pull/1499

And to some extent the precise definition will be up to the squads. So I don't really have answer for the question. 

I would expect that this isn't breaking change.

Comment (ifakhrutdinov) [reply Technical-Steering-Committee/projects.md:79]: Thanks for the clarification.

**Pull request merged**